### DISCUSSION TITLE
Error retrieving images

### DISCUSSION THREAD

**Discussion by Jacielouv** [kvos/CoastSat#548] [Q&A] [answered]
Hi, i' having some issues with CoastSata while downloading images. It works fine with some of the first imaged, but there is a moment when it gives me this error:

![image](https://github.com/user-attachments/assets/00ea8566-a3f7-42b7-a77c-aef07452ff4a)

Maybe it has to be linked to the variable srcDSTab in "return wrapper_GDALWarpDestName(destNameOrDestDS, srcDSTab, opts, callback, callback_data)", since this variable is [None].

I have already used CoastSat previously with no problems, so I don't understand at all what's happening here.

Hope anyone can help.

**Comment by kvos** [comment]
strange, I have never seen this GDAL error. Which version of gdal are you running? try to update to 2.9.

**Comment by Jacielouv** [comment, marked as answer]
I think that i had some conflict with other packages. I created a new environment and it works fine now.
